autoload/lsp.vim

```diff
@@ -548,12 +551,19 @@ function! lsp#default_get_supported_capabilities(server_info) abort
     \           'dynamicRegistration': v:false,
     \           'linkSupport' : v:true
     \       },
+    \       'publishDiagnostics': {
+    \           'relatedInformation': v:true,
+    \       },
     \       'rangeFormatting': {
     \           'dynamicRegistration': v:false,
     \       },
     \       'references': {
     \           'dynamicRegistration': v:false,
     \       },
+    \       'rename': {
+    \           'dynamicRegistration': v:true,
+    \           'prepareSupport': v:true
+    \       },
```

Comment:
ditto for dynamicRegistration.

I think we also should add the following capabilities.

```
prepareSupportDefaultBehavior: 1
```